src/controllers/edgeconnect/controller.go

```diff
@@ -0,0 +1,124 @@
+package edgeconnect
+
+import (
+	"context"
+	"os"
+	"time"
+
+	edgeconnectv1alpha1 "github.com/Dynatrace/dynatrace-operator/src/api/v1alpha1/edgeconnect"
+	"github.com/Dynatrace/dynatrace-operator/src/controllers/dynakube/version"
+	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
+	"github.com/Dynatrace/dynatrace-operator/src/kubesystem"
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	defaultUpdateInterval = 30 * time.Minute
```

Comment:
where did the others go?
```
	errorUpdateInterval   = 1 * time.Minute
	changesUpdateInterval = 5 * time.Minute
```

Reply:
Added `errorUpdateInterval`, `changesUpdateInterval` is not yet useful, except if we use it after updating the status timestamp which is done basically everytime now, so it would replace `defaultUpdateInterval`